libs/designsystem/src/lib/components/text-link/text-link.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'kirby-text-link',
+  templateUrl: './text-link.component.html',
+  styleUrls: ['./text-link.component.scss'],
+})
+export class TextLinkComponent implements OnInit {
+  @Input() link: string;
+  @Input() text: string;
+
+  constructor() {}
+  ngOnInit(): void {}
+
+  checkDomain(url) {
+    if (url.indexOf('//') === 0) {
+      url = location.protocol + url;
+    }
+    return url
+      .toLowerCase()
+      .replace(/([a-z])?:\/\//, '$1')
+      .split('/')[0];
+  }
+
+  isExternal() {
```

Comment:
Consider moving to `ngOnChanges` method to only evaluate link when it's set/updated...